test/groovy/util/JenkinsLoggingRule.groovy

```diff
@@ -0,0 +1,37 @@
+package util
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class JenkinsLoggingRule implements TestRule {
+
+    final BasePipelineTest testInstance
+
+    String log = ""
```

Comment:
not sure if we should have a list here. I do not say that a list is better, just a hint.